starknet_py/tests/e2e/fixtures/contracts.py

```diff
@@ -93,6 +95,54 @@ async def deploy_contract(account: BaseAccount, class_hash: int, abi: List) -> C
     return deployment_result.deployed_contract
 
 
+async def deploy_v1_contract(
+    account: BaseAccount, contract_file_name: str, calldata: Dict[str, Any]
```

Comment:
Nitpick, but perhaps passing whole paths to this method would be a better and more explicit approach.

Reply:
I'm not sure, since we want to keep cairo1 contracts in the one repository (although for now)